src/test/java/org/commcare/formplayer/tests/ScreenSizeTest.java

```diff
@@ -0,0 +1,102 @@
+package org.commcare.formplayer.tests;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.when;
+
+import com.google.common.collect.ImmutableMultimap;
+
+import org.commcare.formplayer.beans.NewFormResponse;
+import org.commcare.formplayer.beans.menus.EntityListResponse;
+import org.commcare.formplayer.beans.menus.QueryResponseBean;
+import org.commcare.formplayer.objects.QueryData;
+import org.commcare.formplayer.objects.SerializableFormSession;
+import org.commcare.formplayer.session.FormSession;
+import org.commcare.formplayer.utils.FileUtils;
+import org.commcare.formplayer.utils.MockRequestUtils;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.cache.CacheManager;
+
+import java.util.Hashtable;
+
+@WebMvcTest
+public class ScreenSizeTest extends BaseTestClass {
```

Comment:
Don't think we need this test anymore as it test the same thing as `testWindowWidthParam` with unnecessary setup.